index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, intial-scale=1.0"/>
+    <title>First Portfolio</title>
+    <link href="./style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!-- START NAVBAR SECTION-->
+    <nav>
+        <div class="navbar">
+            <img src="./img/image_geometry_1.svg" alt="image1"/>
+            <button type="button" class="nav-btn">
+                <div class="lines line1"></div>
+                <div class="lines line2"></div>
+                <div class="lines line3"></div>
+            </button>
+        </div>
+    </nav>
```

Comment:
I kindly recommend wrapping the navbar with the semantic HEADER tag, this is good practice in web development.